src/main/scala/com/spark/practice/students/harsha/sparkBatch/SparkBatchCsvReader.scala

```diff
@@ -0,0 +1,80 @@
+package com.spark.practice.students.harsha.sparkBatch
```

Comment:
Don't know why this file has been committed. Don't want to commit this, Any suggestion to delete this file from PR?

Reply:
delete this file from your branch and commit it